Gemfile

```diff
@@ -50,6 +51,7 @@ group :development, :test do
   gem "bundler-audit"
   gem "climate_control"
   gem "codeclimate-test-reporter"
+  gem "dotenv-rails"
```

Comment:
If you try to access some of these envs through `Rails.application.secrets` then this does not work.

Reply:
in which environment? I was testing Twilio stuff accessed via `Rails.application.secrets` and I believe it worked in dev, staging, and prod.

Reply:
@jayroh - Foreman automatically loads the `.env` file, if you use foreman to run the app locally everything should work for yah.

Reply:
@zspencer @jessieay: What if you're *not* using foreman? I often don't use foreman locally for debugging reasons -- foreman + pry don't mix well.

Reply:
Ahh, in that case you can prefix your commands with `dotenv` which will load the `.env` file. To be honest though, I don't mind having dotenv-rails as a development/test dependency because it's quite light and (I think?) brings no additional dependencies into the mix.

Reply:
ah ok -- we install foreman in the setup script and tell people to run foreman in the CONTRIBUTING doc so I assumed we were all using foreman. Not problematic from my side to include here, perhaps just give a tiny comment about it being useful for running server without foreman?

Reply:
> perhaps just give a tiny comment about it being useful for running server without foreman

@jessieay - where would you expect/like to see a comment wrt this? Because I can definitely do that.  In the commit? In CONTRIBUTING? Just want to make sure I get it right.

Reply:
just right here in the gemfile next to the gem works for me!